scvi/model/base/_base_model.py

```diff
@@ -32,9 +32,7 @@ class BaseModelClass(ABC):
     def __init__(self, adata: Optional[AnnData] = None):
         if adata is not None:
             if "_scvi" not in adata.uns.keys():
-                raise ValueError(
-                    "Please setup your AnnData with scvi.data.setup_anndata(adata) first"
-                )
+                raise ValueError("Please set up your AnnData with setup_anndata first")
```

Comment:
Would it be possible to do something like __name__.setup_anndata() to customize the message for the model class?